blocks/extensions/saveload.py

```diff
@@ -119,49 +120,28 @@ class LoadFromDump(TrainingExtension):
 
     """
     def __init__(self, state_path, **kwargs):
-        super(LoadFromDump, self).__init__(**kwargs)
-        self.manager = MainLoopDumpManager(state_path)
+        super(Load, self).__init__(**kwargs)
+        self.state_path = state_path
+
+    def load(self):
+        with open(self.state_path, "rb") as source:
+            return load(source)
+
+    def load_to(self, main_loop):
+        loaded_main_loop = self.load()
```

Comment:
I should say that this new `Load` extension is less robust than `LoadFromDump`, because the main loop, including all the bricks, has to be unpicklable. It is often the case, that one simply can not do this, because they are working already with a new code version.